### PR TITLE
Updated and added photo_reference to show images on the fav-restaurant page

### PR DESCRIPTION
**Pull request by Kimberly-Rodriguez**

### CODE REVIEW

File: server/models/User.js [hunk 1]
```diff
@@ -49,6 +49,9 @@ const userSchema = new Schema({
         price_level: {
           type: Number,
         },
+        photo_reference: {
+          type: String,
+        },
```
Comment (johndavis92790) [inline server/models/User.js:54]: You can also add the other data points from the FavRestaurantData interface here if you want: lat: number, lng: number, rating: number, types: [string].

**Review by johndavis92790 — CHANGES_REQUESTED** [review]

File: server/models/User.js (hunk 1 above)
Comment (Kimberly-Rodriguez) [reply server/models/User.js:54]: I agree. Changes made, please let me know what you think.

File: server/models/User.js [hunk 2]
```diff
@@ -49,6 +49,21 @@ const userSchema = new Schema({
...
+        lng: {
+          type: Number,
+        },
+        rating: {
+          type: Number,
+        },
+        types: {
+          type: String,
+        },
```
Comment (johndavis92790) [inline server/models/User.js:66]: The `type` of `types` should be `[String]`, not just `String` because it's an array of Strings.

**Review by johndavis92790 — CHANGES_REQUESTED** [review]

**Review by robyng — APPROVED** [review]
looks good

**Review by johndavis92790 — APPROVED** [review]

**Pull request merged**